christfellowshipchurch/apollos-api
# Display error message when leader changes own status

### About
This PR changes group leaders permissions to not be able to change their own status in a group.

### Test Instructions
Go to a group where you're a leader and try changing your own status, it should display an error/modal as seen on screenshot below.

### Screenshots 
<img width="1440" alt="Screen Shot 2022-01-05 at 3 55 04 PM" src="https://user-images.githubusercontent.com/46769629/148288150-49ef0ddd-3c03-44e5-ba67-f8686c42b5b4.png">

<img width="382" alt="Screen Shot 2022-01-05 at 3 55 11 PM" src="https://user-images.githubusercontent.com/46769629/148288154-6b80df4e-e400-4339-b9ee-fb3131a04f76.png">

### Closes Tickets
[CFDP-1913](https://christfellowshipchurch.atlassian.net/browse/CFDP-1913)

### Related Tickets
N/A
